misaligned.py

```diff
@@ -1,13 +1,17 @@
-
+reference_manual = []
 def print_color_map():
+    global reference_manual
     major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
     minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
     for i, major in enumerate(major_colors):
         for j, minor in enumerate(minor_colors):
             print(f'{i * 5 + j} | {major} | {minor}')
+            reference_manual.append(f'{i * 5 + j} | {major} | {minor}')
     return len(major_colors) * len(minor_colors)
 
 
 result = print_color_map()
-assert(result == 25)
+assert(result == 24)

```

Comment:
result == 24 is a wrong expectation. The correct expectation is value = num of major colors * num of minor colors.